docs/storage/clone_api.md

```diff
@@ -155,28 +157,18 @@ After the clone is finished, the target can be inspected:
 kubectl get vm vm-clone-target -o yaml
 ```
 
-## Future roadmap
-
-The clone API is in an early alpha version and may change dramatically. There are many improvements and features
-that are expected to be added, the most significant goals are:
-
-* Add more supported source types like `VirtualMachineInstace` in the future.
-* Add a cross-namespace clone support. This needs to be supported for snapshots / restores first.
```

Comment:
are this 2 goals not expected to happen anymore?

Reply:
> Add more supported source types like `VirtualMachineInstace` in the future.

Not really relevant anymore now that the snapshot source is supported.

> Add a cross-namespace clone support. This needs to be supported for snapshots / restores first.

As written, it has to be supported by snapshot/restore first, and I'm not aware of a concrete plan to support it. Perhaps in the further future.